frontend/packages/dev-console/src/components/import/advanced/ServerlessScalingSection.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { NumberSpinnerField } from '../../formik-fields';
+import FormSection from '../section/FormSection';
+
+const ServerlessScalingSection: React.FC = () => {
+  return (
+    <FormSection title="Scaling">
+      <NumberSpinnerField
+        name="serverless.scaling.minpods"
+        label="Min Pods"
+        helpText="The lower limit for the number of pods that can be set by autoscaler. If not specified defaults to 0."
+      />
+      <NumberSpinnerField
+        name="serverless.scaling.maxpods"
+        label="Max Pods"
+        helpText="The upper limit for the number of pods that can be set by autoscaler."
+      />
+      <NumberSpinnerField
+        name="serverless.scaling.concurrencyrequest"
+        label="Concurrency Request"
```

Comment:
@serenamarie125 should this be `Concurrency Target` ?

Reply:
@parvathyvr ^^

Reply:
@invincibleJai @christianvogt  I confirmed with @markusthoemmes  that it will called 'Concurrency Target'

Reply:
done